bootstrap.sh

```diff
@@ -7,7 +7,7 @@ if [[ -z "${PREFIX}" ]]; then
 fi
 
 if [[ -z "${RELEASE}" ]]; then
-  RELEASE=master
+  RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/crossplaneio/crossplane-cli/releases/latest | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
```

Comment:
This is awesome - great idea! I tested this locally and discovered that `latest` doesn't include releases marked as pre-releases, so we'll want to keep that in mind when we're cutting our releases.